.circleci/config.yml

```diff
@@ -58,4 +58,4 @@ workflows:
       - test:
           matrix:
             parameters:
-              version: ["latest", "1.15", "1.14", "1.13"]
+              version: ["1.15", "1.14", "1.13"]
```

Comment:
change discussed offline with @adamiklukasz - will be brought back by his changes